locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -5,181 +5,223 @@
 msgstr ""
 "Project-Id-Version: Pronterface jm1\n"
-"POT-Creation-Date: 2014-03-30 14:35+CEST\n"
-"PO-Revision-Date: 2012-01-23 10:01+0100\n"
+"POT-Creation-Date: 2021-01-21 02:35+0100\n"
+"PO-Revision-Date: 2021-01-21 06:05+0100\n"
 "Last-Translator: Christian Metzen <metzench@ccux-linux.de>\n"
 "Language-Team: DE <LL@li.org>\n"
-"Language: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 2.4.2\n"
 
 #: printrun/excluder.py:26
 msgid ""
 "Part excluder: draw rectangles where print instructions should be ignored"
 msgstr ""
+"Part excluder: Zeichnen Sie Rechtecke, bei denen Druckanweisungen ignoriert "
+"werden sollten"
 
 #: printrun/excluder.py:27 printrun/excluder.py:29
 msgid "Reset selection"
-msgstr ""
+msgstr "Auswahl zurücksetzen"
 
-#: printrun/gcodeplater.py:43 printrun/gcodeplater.py:44
+#: printrun/gcodeplater.py:77 printrun/gcodeplater.py:78
 msgid "GCODE files (*.gcode;*.GCODE;*.g)"
-msgstr ""
-
-#: printrun/gcodeplater.py:112 printrun/gcodeplater.py:152
-msgid "Warning: no rotation support for now, object won't be correctly rotated"
-msgstr ""
+msgstr "GCODE Dateien (*.gcode;*.GCODE;*.g)"
 
-#: printrun/gcodeplater.py:141 printrun/gcodeplater.py:178
+#: printrun/gcodeplater.py:199 printrun/gcodeplater.py:236
 msgid "Exported merged G-Codes to %s"
 msgstr ""
 
-#: printrun/gcview.py:352
+#: printrun/gcview.py:106
+msgid "Invalid layer for injection"
+msgstr "Ungültiger Abschnitt angegeben"
+
+#: printrun/gcview.py:114
+msgid "Invalid layer for edition"
+msgstr "Ungültiger Abschnitt angegeben"
+
+#: printrun/gcview.py:437
 msgid "Fit to plate"
 msgstr ""
 
-#: printrun/gcview.py:353
+#: printrun/gcview.py:438
 msgid "Fit to plate [F]"
 msgstr ""
 
-#: printrun/gui/__init__.py:19 printrun/pronterface.py:45
-msgid "WX is not installed. This program requires WX to run."
-msgstr "WX ist nicht installiert. Dieses Programm erfordert WX zum Starten."
+#: printrun/gcview.py:466
+msgid "Layer %d -%s Z = %.03f mm"
+msgstr ""
+
+#: printrun/gcview.py:468 printrun/gcview.py:497
+msgid "Entire object"
+msgstr "Gesamtes Objekt"
 
-#: printrun/gui/__init__.py:159
+#: printrun/gl/panel.py:172
+msgid "OpenGL failed, disabling it:"
+msgstr ""
+
+#: printrun/gui/__init__.py:23 printrun/pronterface.py:49
+msgid "WX >= 4 is not installed. This program requires WX >= 4 to run."
+msgstr ""
+"WX >=4 ist nicht installiert. Dieses Programm erfordert WX >= 4 zum Starten."
+
+#: printrun/gui/__init__.py:185
 #, fuzzy
 msgid "Commands"
 msgstr "Kommando"
 
-#: printrun/gui/__init__.py:160
+#: printrun/gui/__init__.py:186
 msgid "Status"
 msgstr ""
 
-#: printrun/gui/controls.py:126
+#: printrun/gui/__init__.py:187 printrun/pronterface.py:290
+#: printrun/pronterface.py:627
+msgid "Tabbed with platers"
+msgstr ""
+
+#: printrun/gui/__init__.py:203 printrun/pronterface.py:828
 #, fuzzy
+msgid "Plater"
+msgstr "Heizelement"
```

Comment:
This is definitely not a correct translation - the plater is the part of the application that lets you arrange objects on the printing plate, whereas Heizelement means heater.

Reply:
so do you know a good translation for plater?

Reply:
Not really - best I can think of is "Zusammenstellung" but that's not quite the same thing. But "Heizelement" is definitely wrong so it's better to leave it untranslated if we can't figure out a better word.